packages/router/src/history.tsx

```diff
@@ -7,19 +7,19 @@ const createHistory = () => {
     listen: (listener: Listener) => {
       const listenerId = 'RW_HISTORY_LISTENER_ID_' + Date.now()
       listeners[listenerId] = listener
-      window?.addEventListener('popstate', listener)
+      window.addEventListener('popstate', listener)
```

Comment:
Didn't actually mean to include this in the PR, but maybe it could go in... `window` won't actually ever be `undefined` or anything like that when navigating, right? @dac09 deferring to you.

Reply:
yeah window is safe here, since its inside the listener. Might be a good idea to use `global` anyway just for consistency

Reply:
Speaking of `popstate` does this issue need any consideration? https://github.com/redwoodjs/redwood/issues/1892